src/reanimated2/utils.ts

```diff
@@ -32,3 +32,14 @@ export function isSharedValue<T>(value: any): value is SharedValue<T> {
   'worklet';
   return value?._isReanimatedSharedValue === true;
 }
+
+/**
+ * @param val preferred value
```

Comment:
```suggestion
 * @param value clamped value
```

Reply:
`clamped value` would be the result value, no? Did you mean `value to be clamped`?

Reply:
It's the value you are clamping - clamped value, same as shareable value or animated value

Reply:
But I guess value to he clamped sounds more precise :like: